src/main/java/jenkins/scm/api/SCMSourceOwners.java

```diff
@@ -69,7 +70,7 @@
         public Iterator<SCMSourceOwner> iterator() {
             Jenkins j = Jenkins.getInstance(); // TODO 1.590+ getActiveInstance
```

Comment:
Obviously it is not to be called from slave agent JVMs. My only concern would be calls made during shutdown. I do not _think_ there should be any, since normally this method would be called in the context of an HTTP request handler thread (from an SCM webhook). We _do_ wait for all pending HTTP requests to complete before terminating, right @stephenc? …